pkg/manifestchecker/types.go

```diff
@@ -175,6 +175,20 @@ func (result *ToolValidationResult) NotParseableVersionOutputCommand(msg string)
 	return result
 }
 
+func (result *ToolValidationResult) CheckErrors() []error {
```

Comment:
this function should be just IsValid returning result.IsToolAvailable && result.IsVersionValid && result.IsCommandVersionOutputParseable